layouts/partials/platen/docs/inject/head.html

```diff
@@ -0,0 +1,2 @@
+{{- $platenStyles := resources.Get "platen.scss" | resources.ExecuteAsTemplate "platen.scss" . | resources.ToCSS | resources.Minify | resources.Fingerprint }}
+<link rel="stylesheet" href="{{ $platenStyles.RelPermalink }}" {{ template "integrity" $platenStyles }}>
```

Comment:
We'll need to think about this a little bit - we don't need to _solve_ it here, but if we're going to inject our own stuff into the head, we need to provide theme users with a way to add their own injections without having to shadow ours and copy-paste our code in.

Reply:
That's true. I'm going to aim to refactor this to inject a new file via `baseof` because that one's already non-standard from hugo-book.